provider/k8s.go

```diff
@@ -6,21 +6,29 @@ import (
 	"fmt"
 	"io"
 	"os"
+	"time"
+	// "io/ioutil"
+	"bytes"
 	"os/exec"
+	"reflect"
 	"strings"
 
+	"github.com/featureform/helpers"
 	"github.com/featureform/kubernetes"
 
+	parquet "github.com/segmentio/parquet-go"
 	"go.uber.org/zap"
 	"gocloud.dev/blob"
+	azureblob "gocloud.dev/blob/azureblob"
+	_ "gocloud.dev/blob/fileblob"
 	_ "gocloud.dev/blob/memblob"
 )
 
 type K8sOfflineStore struct {
 	executor Executor
 	store    BlobStore
 	logger   *zap.SugaredLogger
-	query    *defaultOfflineSQLQueries
+	query    *defaultSparkOfflineQueries
```

Comment:
any reason why the type is `defaultSparkOfflineQueries`?

Reply:
the spark queries are the default queries converted to work with the spark dataframes go converted language, which doesn't allow aliasing without nesting